docs/data/xrootd-authorization.md

```diff
@@ -0,0 +1,141 @@
+to authenticate and authorize users based on X509 certificates and VOMS attributes. Perform the following instructions
+on all data nodes:
+
+1. Install [CA certificates](/common/ca#installing-ca-certificates) and [manage CRLs](/common/ca#installing-ca-certificates#managing-certificate-revocation-lists)
+
+1. Follow the instructions for requesting a [service certificate](/security/host-certs#requesting-and-installing-a-service-certificate),
+   using `xrootd` for both the `<SERVICE>` and `<OWNER>`, resulting in a certificate and key in `/etc/grid-security/xrootd/xrootdcert.pem`
+   and `/etc/grid-security/xrootd/xrootdkey.pem`, respectively.
+
+1. Install and configure the [LCMAPS VOMS plugin](/security/lcmaps-voms-authentication)
+
+1. Install `xrootd-lcmaps` and necessary configuration:
+
+        :::console
+        root@host # yum install xrootd-lcmaps vo-client
+
+1. Configure access rights for mapped users by creating and modifying the XRootD [authorization file](#authorization-file)
+
+1. Modify your XRootD configuration:
+
+    1. Choose the configuration file to edit based on the following table:
+
+        | If you are running XRootD in... | Then modify the following file...   |
+        |:--------------------------------|:------------------------------------|
+        | Standalone mode                 | `/etc/xrootd/xrootd-standalone.cfg` |
+        | Clustered mode                  | `/etc/xrootd/xrootd-clustered.cfg`  |
+
+    1. Add the following lines to the configuration that you chose above:
+
+            xrootd.seclib /usr/lib64/libXrdSec-4.so
+            sec.protocol /usr/lib64 gsi -certdir:/etc/grid-security/certificates \
+                                -cert:/etc/grid-security/xrootd/xrootdcert.pem \
+                                -key:/etc/grid-security/xrootd/xrootdkey.pem -crl:1 \
+                                -authzfun:libXrdLcmaps.so -authzfunparms:--loglevel,0,--policy,authorize_only \
+                                -gmapopt:10 -gmapto:0
+            acc.authdb /etc/xrootd/auth_file
+            ofs.authorize
+
+1. Restart the [relevant services](#using-xrootd)
+
+To verify the LCMAPS security, run the following commands from a machine with your user certificate/key pair,
+`xrootd-client`, and `voms-clients-cpp` installed:
+
+1. Destroy any pre-existing proxies and attempt a copy to `<DESTINATION PATH>` on the `<XROOTD HOST>` to verify failure:
+
+        :::console
+        user@client $ voms-proxy-destroy
+        user@client $ xrdcp /bin/bash root://%RED%<XROOTD HOST>%ENDCOLOR%/%RED%<DESTINATION PATH>%ENDCOLOR%
+        180213 13:56:49 396570 cryptossl_X509CreateProxy: EEC certificate has expired
+        [0B/0B][100%][==================================================][0B/s]
+        Run: [FATAL] Auth failed
+
+1. On the XRootD host, add your DN to [/etc/grid-security/grid-mapfile](/security/lcmaps-voms-authentication#mapping-users)
```

Comment:
Should there be a step for ensuring they're authorized to write to the destination path in the XRootD authorization file?

Reply:
Just discussed this with @matyasselmeci: the answer is yes, we need two steps:

1. Add a line to the XRootD config to make sure that the mapped user can write to `<DESTINATION PATH>`
1. Restart the xrootd service <link to managing xrootd section of the other doc>

Reply:
Updated.